### ISSUE TITLE
slow start up and performance

### ISSUE THREAD

**Issue by rawlingsj** [sonatype/docker-nexus3#25]
I'm looking to migrate to nexus 3 from 2.x using the OSS docker image but I'm experiencing some performance issues.  Just a simple `docker run -ti  -p 8081:8081 sonatype/nexus3:3.0.0` is taking ~7 mins for Nexus to be ready.  We start up a lot of clusters and local dev environments that include Nexus OOTB so I'm wondering if there's any suggestions on how we can improve the start up speed?

We use Kubernetes so have control over the resources we can allocate, what are the CPU + memory recomendations if we want to use Nexus for java, node and docker images?

Also builds are taking much much longer, and keep failing with errors like

```
[ERROR] Failed to execute goal on project micro: Could not resolve dependencies for project org.example:micro:jar:1.12.12: Failed to collect dependencies {..etc etc...} Could not transfer artifact com.squareup.okhttp3:okhttp:pom:3.4.1 from/to nexus (http://nexus/repository/public/): 
```

If I restart the build then it will get further but then fail on another dependency.  I'll start to investigate more, get some real numbers for comparison and look at the resources used but I just wanted to raise the issue initially incase others are seeing similar performance issues.  

<img width="958" alt="screenshot 2016-09-04 18 56 36" src="https://cloud.githubusercontent.com/assets/1457180/18232917/d6e87558-72d1-11e6-9161-437b35adf9bc.png">

**Comment by DarthHater** [comment]
Moved to: https://issues.sonatype.org/browse/NEXUS-12861

**Comment by peterlynch** [comment]
Please try Nexus 3.3 - there were many performance issues resolved in 3.3.

If a problem remains in 3.3 that is identical, please continue the discussion at https://issues.sonatype.org/browse/NEXUS-12861
